src/test/java/com/bamdoliro/maru/presentation/question/QuestionControllerTest.java

```diff
@@ -0,0 +1,124 @@
+                                fieldWithPath("content")
+                                        .type(JsonFieldType.STRING)
+                                        .description("1024글자 이내의 내용")
+                        )
+                ));
+    }
+
+    @Test
+    void 자주묻는질문을_수정한다() throws Exception {
+        willDoNothing().given(createQuestionUseCase).execute(any(CreateQuestionRequest.class));
+        UpdateQuestionRequest request = new UpdateQuestionRequest("이거 맞나", "아님 말고...");
+
+        User user = UserFixture.createUser();
+        given(jwtProperties.getPrefix()).willReturn("Bearer");
+        given(tokenService.getEmail(anyString())).willReturn(user.getEmail());
+        given(authDetailsService.loadUserByUsername(user.getEmail())).willReturn(new AuthDetails(user));
+
+        long id = 1;
+        mockMvc.perform(put("/question/{id}", id)
+                        .header(HttpHeaders.AUTHORIZATION, AuthFixture.createAuthHeader())
+                        .accept(MediaType.APPLICATION_JSON)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(toJson(request)))
+                .andDo(restDocs.document(
+                        requestHeaders(
+                                headerWithName(HttpHeaders.AUTHORIZATION)
+                                        .description("Bearer token")
+                        ),
+                        pathParameters(parameterWithName("id").description("질문 ID")),
+                        requestFields(
+                                fieldWithPath("title")
+                                        .type(JsonFieldType.STRING)
+                                        .description("64글자 이내의 제목"),
+                                fieldWithPath("content")
+                                        .type(JsonFieldType.STRING)
+                                        .description("1024글자 이내의 내용")
+                        )
+                ));
+    }
+    @Test
+    void 자주묻는질문을_수정할_때_자주묻는질문이_없으면_에러가_발생한다() throws Exception {
+
+        UpdateQuestionRequest request = new UpdateQuestionRequest("이거 맞나", "아님 말고...");
+        willThrow(new QuestionNotFoundException()).given(updateQuestionUseCase).execute(eq(1L), any(UpdateQuestionRequest.class));
+
+        User user = UserFixture.createUser();
+        given(jwtProperties.getPrefix()).willReturn("Bearer");
+        given(tokenService.getEmail(anyString())).willReturn(user.getEmail());
+        given(authDetailsService.loadUserByUsername(user.getEmail())).willReturn(new AuthDetails(user));
+
+        long id = 1;
+        mockMvc.perform(put("/question/{id}", id)
+                        .header(HttpHeaders.AUTHORIZATION, AuthFixture.createAuthHeader())
+                        .accept(MediaType.APPLICATION_JSON)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(request)))
```

Comment:
```suggestion
                        .content(toJson(request)))
```

상위 상위 클래스에 toJson 메서드가 있어서 그거 쓰시면 댐니다